x-pack/test/functional/apps/ml/data_visualizer/file_data_visualizer.ts

```diff
@@ -112,6 +112,47 @@ export default function ({ getService }: FtrProviderContext) {
         fieldNameFiltersResultCount: 1,
       },
     },
+    {
+      suiteSuffix: 'with a file containing geo field',
+      filePath: path.join(__dirname, 'files_to_import', 'geo_file.csv'),
+      indexName: 'user-import_1',
```

Comment:
The destination index for the import is deleted between tests, but I think it would still be good to not use the same index name in the two tests of this suite, but rather change this one to `user-import_2`.